src/details/MarketplaceLink.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import defaultBook from './book-cover.png';
+
+class MarketplaceLink extends Component {
+
+    createMarkup() {
+        return {__html: this.props.link};
+    }
+
+    render() {
+        if (this.props.link) {
+            return(
+                <div className="amazon-link">
+                    <div dangerouslySetInnerHTML={this.createMarkup()} />
```

Comment:
There is a reason this is called "dangerouslysetinnerhtml."

Create templates. Fill with JSON.